lib/LANraragi/Model/Tankoubon.pm

```diff
@@ -68,6 +73,12 @@ sub create_tankoubon( $name, $tank_id ) {
                 $isnewkey = 1;
             }
         }
+    } else {
+        # Get name
+        my @name = $redis->zrangebyscore($tank_id, 0, 0, qw{LIMIT 0 1});
+        my $n = redis_decode($name[0]);
+
+        $redis->zrem($tank_id, $n);
```

Comment:
There is, it's outside the else, because setting the name is an step that has to be done even if the list is new. The else is just to delete the old one(And I changed the name to @old_name).

Reply:
oh woops it was hidden in the previous diff by github my bad